src/components/PlioQuestion.vue

```diff
@@ -205,10 +214,10 @@ export default {
       // Show highlighted options if coming back to an answered question
       // Wait 200 ms because it takes some time to find the DOM elements
       if (this.plioQuestion.state == "answered") {
-        setTimeout(() => {
+        requestAnimationFrame(() =>{
```

Comment:
This eliminates the need for keeping a 200ms `setTimeOut`. More details [here](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)

Although this is used to handle animations, it works as a hack in a lot of cases where we need to repaint the screen and we ask the browser to finish all its tasks and go into this block only when everything is updated/painted on the screen.

Reply:
<img width="780" alt="Screenshot 2021-02-28 at 12 01 01 PM" src="https://user-images.githubusercontent.com/13185699/109409922-a260d680-79bc-11eb-9d61-9683cba58dff.png">
this seems to indicate that this function is called before the repaint?

Reply:
I think the argument (no argument given here) is the function that will be called before the repaint, and the method itself will be called after

Reply:
Actually, reading more, I'm not 100% sure this works. It waits for the next screen repaint, which could actually be before eerything is loaded, for example in the middle of an animation at some frame

Reply:
`requestAnimationFrame` can be triggered at the next repaint, not the end of the DOM loading. Unless I'm understanding something wrong!

Reply:
Should I change it back to a setTimeout?
I tested it extensively, it's working in mobile and laptop

Reply:
I don't know. Maybe I'm wrong :D if you are sure what `requestAnimationFrame` does, you can merge! But also be sure.